### DISCUSSION TITLE
How can I make a WEBPUSHBUTTON Momentary ?

### DISCUSSION THREAD

**Discussion by gbananachewicz** [arendst/Tasmota#16205] [Q&A] [answered]
How can I make a WEBPUSHBUTTON Momentary ?
Must I use Rules Bery or something ? 
Its very complicated for very easy function.  
I want form www restart relay2 for 5 seconds. (OFF FOR 5s. then ON just from www) 

PS. SOMEONE or Me can push TOOGLE that make OFF and then connection to localisation will be lost

**Comment by barbudor** [comment, marked as answer]
Maybe you want to use [Pulsetime](https://tasmota.github.io/docs/Commands/#pulsetime) command ?

**Reply by sfromis** [reply]
Browser buttons are simply not momentary, but depends on only taking action when you release the mouse button again. You may be able to cobble up some JavaScript code with Berry to play tricks, but that is way outside of what Tasmota provides.

It is not clear what you really want to do, and that "and then connection to localisation will be lost" is not understandable to me. If what you want is off for 5 seconds, and then automatically back on, then `PulseTime` (as suggested by barbudor) along with `PowerOnState 5` may be useful.
